plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -337,6 +342,87 @@ public void testPredicatePushdown()
                 .isFullyPushedDown();
     }
 
+    @Test
```

Comment:
Is this test copied?

Reply:
it's taking partially (for string columns) from BaseMySqlConnector.testPredicatePushdown and SqlServerConnectorTest